app/keepers/keepers.go

```diff
@@ -106,7 +106,8 @@ import (
 )
 
 const (
-	AccountAddressPrefix = "composable"
+	AccountAddressPrefix = "composable"                                  //?should we change this to pica?
```

Comment:
is it it used? i doubt - just delete. 
basically typical lack of unused code lint in ci i guess....